scripts/bootstrap_history.py

```diff
@@ -266,7 +267,7 @@
 
 - [ ] **Ensure Tool Tests use Latest Release**
 
-    - [ ]  Update GALAXY_RELEASE in https://github.com/galaxyproject/tools-iuc/blob/master/.travis.yml#L6
+    - [ ]  Update GALAXY_RELEASE in https://github.com/galaxyproject/tools-iuc/blob/master/.github/workflows/pr.yaml
```

Comment:
Maybe move the section under **Deploy and Test Release** ?

Reply:
I just merged it. I think updating both workflows is the simplest solution. But one could argue the the PR workflow should be updated after the release in order to spare tool developers from potential bugs in pre-release Galaxy.